src/commands/bso/givetgb.ts

```diff
@@ -13,6 +13,7 @@ export default class extends BotCommand {
 	}
 
 	async run(msg: KlasaMessage, [user]: [KlasaUser]) {
+		if (user.id === msg.author.id) throw "You can't give boxes to yourself!";
```

Comment:
its a minor thing but, we always `return msg.channel.send` now and not throw, for things like this.

Reply:
Fixed it and also fixed the place I copied that from.